TimeTracker/TTATimeTracker.h

```diff
@@ -0,0 +1,24 @@
+//
+//  TTATimeTracker.h
+//  TimeTracker
+//
+//  Created by Dillon P on 4/12/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface TTATimeTracker : NSObject
+
+@property (nonatomic) NSString *client;
+@property (nonatomic) NSString *summary;
```

Comment:
We should mark there with `copy` since they may be mutable.
```suggestion
@property (nonatomic, copy) NSString *client;
@property (nonatomic, copy) NSString *summary;
```